web3/_utils/method_formatters.py

```diff
@@ -297,7 +297,9 @@ def apply_list_to_array_formatter(formatter: Any) -> Callable[..., Any]:
         )
     ),
     "transactionsRoot": apply_formatter_if(is_not_null, to_hexbytes(32)),
-    "withdrawals": apply_formatter_to_array(withdrawal_result_formatter),
+    "withdrawals": apply_formatter_if(
+        is_not_null, apply_formatter_to_array(withdrawal_result_formatter)
```

Comment:
I actually noticed this will return an empty tuple if it is empty... I think we need a bugfix to make this formatter use `apply_list_to_array_formatter()`:


```py
    "withdrawals": apply_formatter_if(
         is_not_null, apply_list_to_array_formatter(withdrawal_result_formatter),
     ),
```

This is what the transaction and uncle formatters use. This might be a relevant PR to sneak this into.

Reply:
Oh I do see that with geth, hmm. I was messing around with a rough draft of what `eth-tester` would look like plugged into the latest `py-evm`. I think I'd still rather have the list-like formatter for consistency (which I see was already added) but I think it's good to know it likely won't affect the user base too much.

Reply:
Ah, gotcha. Yeah, I think it makes sense to have it in there in case it comes back as a tuple.